### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by linkittina**

### CODE REVIEW

File: W3schools_ Test.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+I want to compare the small and the large view of the W3schools website
+So that I am going to see the most important differences
+
+Background:
+Given the W3schools page is opened
+Then the URL of the page should be the following:
+|https://www.w3schools.com/|
+And the green navigation bar should be present
+
+Scenario: 1. Checking the layout and using a main function on small
```
Comment (sorosz89) [inline W3schools_ Test.txt]: What do you mean by small?

> 1. Checking the UI elements using the main functions on phone layout

File: W3schools_ Test.txt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+As a tester 
+I want to compare the small and the large view of the W3schools website
+So that I am going to see the most important differences
+
+Background:
+Given the W3schools page is opened
+Then the URL of the page should be the following:
+|https://www.w3schools.com/|
+And the green navigation bar should be present
+
```
Comment (sorosz89) [inline W3schools_ Test.txt]: I plan to use @phone cucumber tag later on. Please insert it. :)

File: W3schools_ Test.txt [hunk 3]
```diff
@@ -0,0 +1,73 @@
+Feature: W3schools layout differences 
+As a tester 
+I want to compare the small and the large view of the W3schools website
+So that I am going to see the most important differences
+
+Background:
+Given the W3schools page is opened
+Then the URL of the page should be the following:
+|https://www.w3schools.com/|
```
Comment (sorosz89) [inline W3schools_ Test.txt]: I think you don't need to use data tables in case of only one item

File: W3schools_ Test.txt [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+And the green navigation bar should be present
+
+Scenario: 1. Checking the layout and using a main function on small
+
+# Checking the home page
+And the example boxes should not be displayed
+And the green navigation menu items should be within the hamburger menu
+And the left white menu should not be displayed
+And the text "THE WORLD'S LARGEST WEB DEVELOPER SITE" should not be displayed
```
Comment (sorosz89) [inline W3schools_ Test.txt]: My recommendation is to use Example table. What do you think about it?

File: W3schools_ Test.txt [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+# Checking the green navigation menu items
+When the hamburger menu icon on the navigation bar is clicked
+Then the green navigation menu should have the following items:
+|TUTORIALS|
+|REFERENCES|
+|EXAMPLES|
+And there should be a close button at the top-right corner
+
+# Using the green navigation menu items
+When the link "TUTORIALS " is clicked
```
Comment (sorosz89) [inline W3schools_ Test.txt]: Please implement something like this to handle the examples:

```
Scenario Outline: Eating
  Given there are <start> cucumbers
  When I eat <eat> cucumbers
  Then I should have <left> cucumbers

  Examples:
    | start | eat | left |
    |  12   |  5  |  7   |
    |  20   |  5  |  15  |
```

File: W3schools_ Test.txt [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+|BOOTSTRAP|
+|HOW TO|
+|JQUERY|
+|W3.CSS|
+|ANGULARJS|
+|MORE|
+|REFERENCES|
+|EXAMPLES|
+
+Scenario: 2. Checking the layout and using a main function on large
```
Comment (sorosz89) [inline W3schools_ Test.txt]: @desktop

... on desktop layout

File: W3schools_ Test.txt [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+# Checking the home page
+And the example boxes should be displayed
+And the green navigation menu items should not be within the hamburger menu
+And the left white menu should be displayed
+And the text "THE WORLD'S LARGEST WEB DEVELOPER SITE" should be displayed
```
Comment (sorosz89) [inline W3schools_ Test.txt]: Scenario Outline + Examples:

File: W3schools_ Test.txt [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+# Checking the green navigation menu items
+And the green navigation menu should have the following items:
+|TUTORIALS|
+|REFERENCES|
+|EXAMPLES|
+And there should be a close button at the top-right corner
+
+# Using the green navigation menu items
+When the link "TUTORIALS " is clicked
```
Comment (sorosz89) [inline W3schools_ Test.txt]: Is the extra spacing intentional? "TUTORIALS "

File: W3schools_ Test.txt (hunk 1 above)
Comment (linkittina) [reply W3schools_ Test.txt]: Fixed.

File: W3schools_ Test.txt (hunk 2 above)
Comment (linkittina) [reply W3schools_ Test.txt]: Fixed.

File: W3schools_ Test.txt (hunk 3 above)
Comment (linkittina) [reply W3schools_ Test.txt]: Fixed.

File: W3schools_ Test.txt (hunk 6 above)
Comment (linkittina) [reply W3schools_ Test.txt]: Fixed.

File: W3schools_ Test.txt (hunk 8 above)
Comment (linkittina) [reply W3schools_ Test.txt]: I missed it when I removed the whitespaces. Fixed. :)

**Pull request merged**